pages/services/beta-cassandra/2.5.0-3.0.16-beta/advanced/index.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: layout.pug
+navigationTitle: Advanced
+excerpt: Advanced features of Apache Cassandra service
+title: Advanced features
+menuWeight: 45
+model: /services/cassandra/data.yml
+render: mustache
+---
+
+#include /services/include/beta-advanced.tmpl
```

Comment:
See note on beta-advanced.tmpl file below. The docs team prefers that you not create new #include files like this, but either use the ones we have or copy/paste the content and edit as necessary.